internal/jobcontainers/utils.go

```diff
@@ -0,0 +1,35 @@
+package jobcontainers
```

Comment:
For the most part, I would prefer we avoid a `utils.go` file, and instead move these functions to the places they are actually used. Generally utils files quickly become dumping grounds for lots of things, which creates disorganization.